src/gd_io_stream.cxx

```diff
@@ -144,6 +138,3 @@ void ostreamIOCtx::FreeCtx(gdIOCtxPtr ctx)
 	{
 	delete (ostreamIOCtx * )ctx;
 	}
-
-#endif /* __cplusplus */
-#endif //ENABLE_CPP_API
```

Comment:
does this mean the file is missing a final newline ?  if so, please add it.  i'm not familiar with this GH output.

Reply:
No; the newline *was* missing previously; that's why it that sign has red background (missing final newline has been removed).